s3watcher/lambda/lib/EnvironmentConfig.ts

```diff
@@ -0,0 +1,32 @@
+interface EnvironmentConfig {
+  stack: string
+  stage: string
+  app: string
+  isDev: boolean
+  region: string
+  profile: string
+  loggingRoleArn?: string
+  loggingStream?: string
```

Comment:
As we're moving to log via cloudwatch, are these props still needed?

Reply:
Nope.